examples/speech_recognition/datasets/asr_prep_json.py

```diff
@@ -21,19 +20,32 @@
 MILLISECONDS_TO_SECONDS = 0.001
 
 
-def process_sample(aud_path, lable, utt_id, sp, tgt_dict):
-    input = {}
-    output = {}
-    si, ei = torchaudio.info(aud_path)
-    input["length_ms"] = int(si.length / si.channels / si.rate / MILLISECONDS_TO_SECONDS)
-    input["path"] = aud_path
+def process_samples(samples, labels, sp_path, tgt_dict):
+    utts = {}
+
+    import sentencepiece as spm
```

Comment:
nit: Seems like there is no way to create json w/o sentencepiece so we will have to import it anyway. Can we import it at the top of the module and fail fast in case if it's not found?

Reply:
This is at a per-process level, so I'm importing it here. We could probably do it at the top as well, but we don't need it in the rest of the script (I removed the above import).